web/template/donor/confirm_person_allowed_to_vouch.gohtml

```diff
@@ -9,27 +9,32 @@
 
       <h1 class="govuk-heading-xl">{{ tr .App "confirmThisPersonIsAllowedToVouchForYou" }}</h1>
 
-      {{ $single := eq 1 (len .Matches) }}
-      {{ if $single }}
-        <p class="govuk-body">
-      {{ else }}
+      {{ if .MultipleMatches }}
         <ul class="govuk-list govuk-list--bullet">
+      {{ else }}
+        <p class="govuk-body">
       {{ end }}
-      
+
+      {{ if $.MultipleMatches }}<li>{{ end }}
```

Comment:
I think the logic is a bit off here - I get an empty bullet when having an attorney and donor with the same name as the voucher:

![Screenshot 2024-06-25 at 11 40 00](https://github.com/ministryofjustice/opg-modernising-lpa/assets/17926619/b3d9f9e6-0f31-4812-9b04-d6887b51f648)

I'm guessing as we are filtering the `MultipleMatches` check on there being only a donor type in the slice?

Reply:
ah yeah, good spot